### PR TITLE
Record IETF 116 decision on handling duplicate suffix tags.

### LINKED ISSUE #25: How to resolve duplicate calendar annotations?  (Error handling.)
BCP-47 takes the first when there is a duplicate:

```javascript
> new Intl.Locale("en-u-ca-buddhist-ca-hebrew").calendar
// buddhist
```

### PR DESCRIPTION
**Pull request by cabo**
Close #25

### CODE REVIEW

**Review by ryzokuken — APPROVED** [review]

File: draft-ietf-sedate-datetime-extended.md [hunk 1]
```diff
@@ -470,6 +464,8 @@ the summer of 2022, it is equivalent to:
 In contrast to this elective use of a suffix tag,
 
     2022-07-08T00:14:07+01:00[!Europe/Paris]
+    2022-07-08T00:14:07Z[!u-ca=chinese,u-ca=japanese]
+    2022-07-08T00:14:07Z[u-ca=chinese,!u-ca=japanese]
```
Comment (ptomato) [inline draft-ietf-sedate-datetime-extended.md:468]: These aren't syntactically valid according to the ABNF — should it be
```
    2022-07-08T00:14:07Z[!u-ca=chinese][u-ca=japanese]
    2022-07-08T00:14:07Z[u-ca=chinese][!u-ca=japanese]
```

File: draft-ietf-sedate-datetime-extended.md [hunk 2]
```diff
@@ -485,6 +481,17 @@ required to reject or perform some other error handling when
 encountering inconsistent or unrecognized suffix tags marked as
 critical.
 
+An application that encounters duplicate use of a suffix key in
+elective suffixes and does not want to perform additional processing
+on this inconsistency MUST choose the first suffix that has that key,
+i.e.,
+
+    2022-07-08T00:14:07Z[u-ca=chinese,u-ca=japanese]
```
Comment (ptomato) [inline draft-ietf-sedate-datetime-extended.md:489]: Ditto,
```
    2022-07-08T00:14:07Z[u-ca=chinese][u-ca=japanese]
```

File: draft-ietf-sedate-datetime-extended.md (hunk 1 above)
Comment (cabo) [reply draft-ietf-sedate-datetime-extended.md:468]: Yes.

File: draft-ietf-sedate-datetime-extended.md (hunk 2 above)
Comment (cabo) [reply draft-ietf-sedate-datetime-extended.md:489]: Indeed.  C&P failure on my part; thanks.

Comment (cabo) [reply draft-ietf-sedate-datetime-extended.md:489]: Now #40 

In Germany, we have a term for the state of mind I must have been in:
"having tomatoes on your eyes".

**Pull request merged**